src/browser/core_state.js

```diff
@@ -499,6 +499,19 @@ function getSocketServerState() {
     return socketServerState;
 }
 
+/**
+ * Get app's very first ancestor
+ */
+function getAppAncestor(descendantAppUuid) {
+    const app = appByUuid(descendantAppUuid);
+
+    if (app && app.appObj && app.appObj.parentUuid) {
```

Comment:
I think @datamadic changed this so the `parentUuid` is now on the `app` level, not on the `appObj` level.